src/core/audio/AudioSdl.cpp

```diff
@@ -64,14 +64,22 @@ AudioSdl::AudioSdl( bool & _success_ful, Mixer*  _mixer ) :
 	m_audioHandle.callback = sdlAudioCallback;
 	m_audioHandle.userdata = this;
 
-  	SDL_AudioSpec actual; 
-
+	SDL_AudioSpec actual; 
```

Comment:
FYI, these SDL2 fixes appear to be redundant with those from #3947. I think it's best to drop a414255d9e55648f93e833c571fac8636c5c5fef from this PR to avoid conflicts.